core/shared/src/main/scala/spire/math/poly/BigDecimalRootRefinement.scala

```diff
@@ -205,15 +205,15 @@ object BigDecimalRootRefinement {
             ExactRoot(lx)
           } else {
             // We try to push lx up a bit to get the sign to change.
-            adjust(lx.add(JBigDecimal.valueOf(1, getEps(lx))), Some(ly), rx, Some(ry))
+            adjust(lx.add(JBigDecimal.valueOf(1, getEps(lx))), None, rx, Some(ry))
```

Comment:
Since we have changed our definition of `lx`, it doesn't make sense to re-use the evaluation of the polynomial at the old definition of `lx`.